src/environments/constants.ts

```diff
@@ -1,4 +1,3 @@
 export const constants = {
   trxRetry: 60, // 1 minute
-  userIdle: 900, // 15 mins in seconds
```

Comment:
Was this value not used previously?

Reply:
Updated. Forgot to revert it back after testing. Thanks for catching this! 👍